packages/material-ui/src/Autocomplete/Autocomplete.js

```diff
@@ -555,6 +556,7 @@ const Autocomplete = React.forwardRef(function Autocomplete(inProps, ref) {
                     title={clearText}
                     className={classes.clearIndicator}
```

Comment:
In the unstyled components we merge classes from `componentsProps` into the already applied ones. Let's make it consistent

```suggestion
                    className={clsx(classes.clearIndicator, componentsProps.clearIndicator?.className)}
```